docs/faq.rst

```diff
@@ -255,13 +255,16 @@ How can I use the Dockstore CLI with Python 2?
 Python 2 support ends in 2020. You can get the Python 2 requirements.txt with ``curl -o requirements.txt "https://dockstore.org/api/metadata/runner_dependencies?python_version=2"``
 but it is currently untested.
 
-How do I add other users as maintainers of a tool?
---------------------------------------------------
+How do I add other users as maintainers of a tool or worklow?
+-------------------------------------------------------------
 
 For tools registered on Quay.io and workflows registered with GitHub,
 Dockstore automatically allows users from the same Quay.io organization
-or GitHub organization to manage tools/workflows together (users will
-need to "Refresh Organization" or "Refresh All").
+or GitHub organization to manage tools and workflows together (users will
```

Comment:
Thanks for updating the screenshot.

I find this paragraph hard to understand. It's one long sentence, talks about tools, workflows and GitHub, but shows a screenshot that is specific to GitHub-based workflows. It doesn't explain when you would use Discover.

It may just be me; I added Natalie as a reviewer for her opinion.